tofu/tests/tests09_tutorials/tests01_runall.py

```diff
@@ -1,25 +1,13 @@
-
-# External modules
 import os
 import shutil
-import types
 import subprocess
-import numpy as np
-import warnings as warn
 import matplotlib.pyplot as plt
-
-# Nose-specific
-from nose import with_setup     # optional
-
-# Importing package tofu.geom
-import tofu as tf
 from tofu import __version__
-
+from pathlib import Path
```

Comment:
Nice, 
This however introduces a new dependency in tofu, except if pathlib is by default in Python 3.6 at least and supported in 3.7 too, is it ?

Reply:
It is, according to its doc, which mentions _New in version 3.4._

Reply:
https://docs.python.org/3/library/pathlib.html